puppetdb/values.yaml

```diff
@@ -2,18 +2,6 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-autosign:
-  psk: ''
```

Comment:
You remove this but it is still used in `puppetdb/templates/secret.yaml`.

I think you should also change said template to only be generated if no existing secret was selected.

This section of the values may be still relevant and require some documentation if you keep the secret template.

Reply:
My bad, i didnt pushed the latest changes.